src/controllers/task.js

```diff
@@ -79,6 +79,39 @@ const TaskController = (taskModel) => {
         })
     })
 
+
+    //my attempt to make a POST request for task-complete:
+    router.post('/complete_task', async (req, res) =>{
+        if (!req.body)
+        return res.status(400).json({
+            message: "Malformed Request"
+        });
+    const body = req.body;
+    const ids = body.ids;
```

Comment:
We should add a check here to make sure that `ids` contains at least 1 element because if I recall correctly, calling `Array.prototype.map()` on an empty array will cause a problem.